tests/tests_ui/test_account.py

```diff
@@ -0,0 +1,30 @@
+import allure
+import pytest
+
+from pages.account_page import AccountPage
+
+
+class TestAccount:
+    @pytest.mark.ui
+    @pytest.mark.account
+    @pytest.mark.smoke
+    @pytest.mark.positive
+    @allure.story('Open profile page')
+    def test_open_profile_page(self, driver, login):
+        login()
+        account_page = AccountPage(driver)
+        with allure.step('Open profile page'):
```

Comment:
Looks like login() is smth that needs to be done in test setup and logout() is smth that needs to be done in test teardown for the Account Page. I suggest to create a fixture in the conftest for the Account page - there do login before test, init the Account page itself and yield it, and at last do the logout. The test could look like:
def test_open_profile_page(self, account_page):
        with allure.step('Open profile page'):
            account_page.open_profile()
            assert account_page.is_profile_open(), 'Profile page is not open.'

Reply:
Done.